snappy/security.go

```diff
@@ -20,207 +20,848 @@
 package snappy
 
 import (
-	"encoding/json"
+	"bytes"
+	"errors"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"os/exec"
 	"path/filepath"
+	"regexp"
 	"strings"
 
-	"gopkg.in/yaml.v2"
-
 	"github.com/ubuntu-core/snappy/dirs"
+	"github.com/ubuntu-core/snappy/helpers"
 	"github.com/ubuntu-core/snappy/logger"
 	"github.com/ubuntu-core/snappy/pkg"
+	"github.com/ubuntu-core/snappy/policy"
+)
+
+type errPolicyNotFound struct {
```

Comment:
by that i mean: keep the type private, but make the members themselves public, e.g. PolKind instead pof polKind.